design-toolkit/components/src/components/notice/index.tsx

```diff
@@ -0,0 +1,298 @@
+        )}
+        {showClose && (
+          <Button
+            color={ButtonColorMap[color]}
+            variant='icon'
+            onPress={onClose}
+          >
+            <Icon>
+              <Cancel />
+            </Icon>
+          </Button>
+        )}
+      </ToastContent>
+    </Toast>
+  );
+}
+
+function matchesMetadata(
+  payload: Record<string, unknown>,
+  metadata?: Record<string, unknown>,
+) {
+  if (!metadata) {
+    return false;
+  }
+
+  return Object.entries(payload).every(
+    ([key, value]) => key in metadata && isEqual(metadata[key], value),
+  );
+}
+
+function NoticeList({
+  id,
+  parentRef,
+  classNames,
+  defaultColor,
+  defaultTimeout,
+  hideClearAll,
+  limit = 3,
+  placement,
+  size = 'medium',
+}: NoticeListProps) {
+  const queue = useMemo(
+    () => new ToastQueue<NoticeContent>({ maxVisibleToasts: limit }),
+    [limit],
+  );
+  const [hasNotices, setHasNotices] = useState(false);
+
+  const { useEmit, useOn } = useBus<
+    NoticeQueueEvent | NoticeDequeueEvent | NoticeActionEvent
+  >();
+  const emitAction = useEmit(NoticeEventTypes.action);
+  const emitActionPrimary = useEmit(NoticeEventTypes.actionPrimary);
+  const emitActionSecondary = useEmit(NoticeEventTypes.actionSecondary);
+  const emitClose = useEmit(NoticeEventTypes.close);
+
+  useOn(NoticeEventTypes.queue, (data) => {
+    if ((id && data.payload.target === id) || !id) {
+      queue.add(
+        {
+          ...data.payload,
+          id: data.payload.id || uuid(),
+          color: defaultColor || data.payload.color,
+        },
+        {
+          timeout: defaultTimeout || data.payload.timeout,
+        },
+      );
+    }
+  });
+
+  useOn(NoticeEventTypes.dequeue, (data) => {
+    if (id && (data.payload as DequeueList).target === id) {
+      queue.clear();
+      return;
+    }
+
+    if (id && (data.payload as DequeueList).target) {
+      return;
+    }
+
+    // @ts-expect-error : queue.queue exists, but is not currently documented
+    const dequeue = queue.queue.filter(
+      (toast: QueuedToast<NoticeContent>) =>
+        ((data.payload as DequeueId).id &&
+          toast.content.id === (data.payload as DequeueId).id) ||
+        ((data.payload as DequeueColor).color &&
+          toast.content.color === (data.payload as DequeueColor).color) ||
+        ((data.payload as DequeueMetadata).metadata &&
+          matchesMetadata(
+            (data.payload as DequeueMetadata).metadata,
+            toast.content.metadata,
+          )),
+    );
+
+    // @ts-expect-error : queue.queue exists, but is not currently documented
+    if (dequeue.length && dequeue.length === queue.queue.length) {
+      queue.clear();
+    } else {
+      for (const toast of dequeue) {
+        queue.close(toast.key);
+      }
+    }
+  });
+
+  useEffect(() => {
+    queue.subscribe(() => {
+      setHasNotices(queue.visibleToasts.length > 0);
+    });
+  });
+
+  return (
+    <PortalProvider parentRef={parentRef}>
```

Comment:
The Notice.List should not default to being portaled to the document root. By default it should render in place without absolute positioning. If the placement prop has been set, then it should portal like how you have it (default of document root, with parentRef override)